docs/architecture/api-credentials-flows.md

```diff
@@ -0,0 +1,18 @@
+# Overview
```

Comment:
Shouldn't `Overview` be of the same heading level as for example `Details`? Same situation in `terminology.md` file.